main.js

```diff
@@ -1,18 +1,29 @@
 // var starredIdeasBtn = document.querySelector('.show-starred-ideas');
 var saveIdeaBtn = document.querySelector('.primary');
 // var favoriteBtn = document.getElementById('favorite');
-// var deleteCardBtn = document.getElementById('deleteCard');
+var deleteCardBtn = document.getElementById('deleteCard');
 // var commentBtn = document.querySelector('.comment-button');
 // var starWhiteImg = document.getElementByName('star-white');
 // var starRedImg = document.getElementByName('star-red');
 // var xWhiteImg = document.getElementByName('x-white');
-// var xRedImg = document.getElementByName('x-red');
+var xRedImg = document.getElementById('x-red');
 var titleInput = document.getElementById('titleInput');
 var bodyInput = document.getElementById('bodyInput');
 var ideaContainerSection = document.querySelector('.idea-container');
 
-
 var savedIdeas = [];
+
+// window.addEventListener("load", getLocalStorage);
+//
+// function getLocalStorage() {
+// for(var i =0; i < localStorage.length; i++){
+//   console.log(localStorage.getItem(localStorage.key(i)))
+// //   savedIdeas.push(JSON.parse(localStorage.getItem(localStorage.key(i))));
+// // savedIdeas.push(JSON.parse
```

Comment:
we should pay attention to line 22,  delete the unnecessary code, it is part duplicated.